docs/UserGuide.md

```diff
@@ -80,17 +80,28 @@ Format: `add n/NAME q/QUANTITY u/UNIT`
 * If the ingredient already exists in the stock, using this command will add to the quantity specified for that 
   ingredient.
 
+Restrictions:
+* Units used must be supported
+* Quantity must be positive
+
 Examples:
 * `add n/Flour q/1 u/kg` adds 1kg of Flour to the stock
 * `add n/Milk q/600 u/g` adds 600ml of Milk to the stock
 
+Supported Units:
+| Unit     | Alias                                 |
+|----------|---------------------------------------|
+| GRAM     | g, gram, GRAM                         |
+| KILOGRAM | kg, kilogram, KILOGRAM                |
+| PIECE    | pc, pcs, piece, pieces, PIECE, PIECES |
+
 ### Using up ingredients : `use`
 
 Depletes a specified amount of an ingredient from stock.
 
 Format: `use n/NAME [q/QUANTITY] [u/UNIT]`
 
-* If no quantity and unit is provided, the entire stock of the specified ingredient will be depleted.
+* If no quantity or unit is provided, the entire stock of the specified ingredient will be depleted.
```

Comment:
I think this part is still and as only when both are missing will it deplete all. Having one missing will lead to invalid command

Reply:
The behaviour is unfortunately inconsistent due to the parsing, as shown in #150 . This was changed if the parsing were to remain the same as the current implementation

Reply:
oh ok then lets stick to this.

Reply:
will merge